plugins/Annotations/API.php

```diff
@@ -334,7 +334,9 @@ public static function getDateRangeForPeriod($date, $period, $lastN = false)
             $endDate = $oPeriod->getDateEnd();
         } else // if the range includes the last N periods
         {
-            list($date, $lastN) = EvolutionViz::getDateRangeAndLastN($period, $date, $lastN);
+            if (!Range::isMultiplePeriod($date, $period)) {
```

Comment:
Maybe a test can be added for this new Annotations API feature?

Reply:
Added